scoring/views.py

```diff
@@ -47,6 +47,10 @@ class LogoutView(LogoutView):
     next_page = "scoring:home"
 
 
+class PrivacyView(TemplateView):
+    template_name = "scoring/privacy.html"
```

Comment:
I _think_ you’ll want to set the `page_title` context variable, so that the `<title>` tag in the `<head>` gets populated?

```python
def get_context_data(self, **kwargs):
    context = super().get_context_data(**kwargs)
    context["page_title"] = "Privacy policy"
    return context
```